CONTRIBUTING.md

```diff
@@ -19,7 +19,7 @@ GNU Make is required.
 
 The [`tools`](tools) directory contains scripts too complex to fit in the
 `Makefile`. Among them, [`tools/pre-push`](tools/pre-push) can be used as a Git
-hook, and covers a large subset (but not all) of CI checks.
+hook, and covers all of the CI checks.
```

Comment:
Revert; this is not true. CI checks run on multiple platforms and can't be easily simulated locally.

Reply:
Hmmm, okay, but it seems like a distinction that isn't particularly important as it runs all the CI checks relevant for the developer's platform. Too, the fact that CI environments typically employ multiple combinations of architectures and OS's is taken as a given by any developer who is not a complete noob. I would argue that saying "and covers all CI checks relevant to your platform" is clearer and more informative.